sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -5,6 +5,8 @@
  * @file az_iot_hub_client.h
  *
  * @brief definition for the Azure IoT Hub client.
+ * @note The IoT Hub MQTT protocol is described at 

```

Comment:
I wonder how `@note` renders in comparison to `@remarks`.

Reply:
Found this answer about the differences: https://stackoverflow.com/questions/34372532/choosing-between-attention-note-remark-todo-and-warning-in-doxygen